apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-index.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MemoryRouter } from 'react-router-dom';
+import { getConfig } from '../../../config';
+import { assertElementWithTestIdOnScreen, renderWithProviders } from '../../../utils/test-utils';
+import { buildMockSuccessfulGETHandler } from '../../../utils/test-utils/buildMockSuccessfulGETHandler';
+import { testContainerComponentErrorHandling } from '../../../utils/test-utils/common-test-cases/test-container-component-error-handling';
+import { setupTestServer } from '../../../utils/test-utils/setupTestServer';
+import { buildMockIndexResponse } from '../../../utils/test-utils/test-data';
+import { TranscribedAudioIndexContainer } from './transcribed-audio-index.container';
+
+const dummyTranscribedAudioItems: ITranscribedAudioViewModel[] = [
```

Comment:
Would it make sense to create more dummy items?  or is that not relevant to this test?

Reply:
It would give us added confidence, provided that we also check for every single item. I'd like to hold off on this until we decide if we are going to share our test data with the backend.